packages/name-controller/src/NameController.test.ts

```diff
@@ -623,8 +631,8 @@ describe('NameController', () => {
 
   describe('updateProposedNames', () => {
     it.each([
-      ['', (controller: NameController) => controller.state.names],
-      [' and no existing type state', () => ({})],
+      ['', {}],
+      [' and no existing type state', { names: {} }],
     ])(
       'creates entry with proposed names if value is new%s',
       async (_, getExistingState) => {
```

Comment:
Nit: `existingState` is probably a better name, now that this is no longer a function